content/docs/components/popover/theming.mdx

```diff
@@ -2,3 +2,174 @@
 id: popover
 scope: theming
 ---
+
+The `Popover` component is a multipart component. The styling needs to be
+applied to each part specifically.
+
+> To learn more about styling multipart components, visit the
+> [Component Style](/docs/styled-system/component-style#styling-multipart-components)
+> page.
+
+## Anatomy
+
+- A: `content`
+- B: `header`
+- C: `body`
+- D: `footer`
+- E: `popper`
+- F: `arrow`
+- G: `closeButton`
+
+## Theming properties
+
+The properties that affect the theming of the `Popover` component are:
+
+- `arrowPadding`: The arrow padding of the popover. Defaults to `8`
+- `gutter`: The distance or margin between the reference and popper. Defautls to
+  `8`
```

Comment:
In this section we want to notice the theming properties that can be overridden that are within the theme. They usually are `variant`, `size` etc... The `arrowPadding` and `gutter` are style props, so they don't belong here. The `Popover` component actually doesn't have any theming properties, it just has a `baseStyle` defined.

Reply:
@nikolovlazar, would it be better to delete this section or to explain that the ```Popover``` doesn't have any theming properties?

Reply:
It's better to explain that there's no defaults to override, but you can always create your own.